.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish Package
```

Comment:
**nit**: I think we can keep it as CI only since we're doing various things here. Had earlier thought we are just doing publish

Reply:
done